Sprint-1/interpret/time-format.js

```diff
@@ -22,3 +23,22 @@ console.log(result);
 // e) What do you think the variable result represents? Can you think of a better name for this variable?
 
 // f) Try experimenting with different values of movieLength. Will this code work for all values of movieLength? Explain your answer
+
+
+/*    Answer
+
+a) 6
+
+b) 1
+
+c) This uses a  modulus operator (%) to find the remainder when movieLength is divided by 60. 
+
+d) Calculating the total number of full minutes in movieLength. 
+movieLength represents the total duration of the movie in seconds ,remainingSeconds is the number of seconds left after accounting for all full minutes (from the previous line).
+By subtracting remainingSeconds from movieLength, we get only the seconds that make up complete minutes, effectively rounding down to the nearest minute.
+
+e)movieDuration
```

Comment:
The original movieLength variable also represented the duration of the movie. What's different about the result variable, and how might we call it to better communicate that?

Reply:
I have updated movieDuration to  timer instead of result.